libs/api/domains/driving-license-book/src/models/studentsTeacherNationalId.response.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class DrivingLicenseBookStudentForTeacher {
+  @Field(() => ID, { nullable: true })
```

Comment:
I might be wrong here ... but this comment goes for the other GraphQL types as well:
Doesn't the schema throw an error if you try to set an ID as nullable?
I assumed this wasn't possible 🤔

Reply:
I don't think there is anything forbidding nullable IDs. I believe any GraphQL field can be null. The schema doesn't seem to enforce non-null on ID, and in fact is omitted for most classes.

Reply:
How would I go about testing your hypothesis?

Reply:
So basically my assumption was that the graphql eslint would notify you that ID's should not be null. But you generally never want to set IDs as nullable, otherwise the in memory cache normalization won't work on the front-end. So you'd rather want the error to propagate.
There are really unique cases when you want the ID to be nullable.

If you want to test it, just try and strip out the ID from the returned result. The cache won't be updated correctly since it won't know how to normalize the returned data -> this is the same reason for why we should always query the GraphQL field's ID